rocketpy/environment/environment.py

```diff
@@ -682,7 +682,7 @@ def set_elevation(self, elevation="Open-Elevation"):
         #     self.elevation = elev
 
         elif self.latitude is not None and self.longitude is not None:
-            self.elevation = self.__fetch_open_elevation()
+            self.elevation = float(self.__fetch_open_elevation())
```

Comment:
Is this to avoid breaking changes regarding the type of `self.elevation`?

Reply:
that was just me trying to debug the problem.

It doesn't really changes the code behavior, we can ignore it.

Reply:
Alright, I am favor rolling back this change here, but this is a non blocking issue for me.